src/main/java/net/snowflake/ingest/streaming/internal/ParquetFlusher.java

```diff
@@ -0,0 +1,271 @@
+package net.snowflake.ingest.streaming.internal;
```

Comment:
There're some comments I made in another PR which would apply here, so things like copyright, no star imports and package private classes, please go through all the newly added files and add them, thanks!